nvautoinstall.spec

```diff
@@ -7,13 +7,17 @@ Summary: NVIDIA Auto Installer for Fedora
 
 License: GPLv3
 Url: https://github.com/t0xic0der/%{srcname}
-Source0: https://github.com/t0xic0der/%{srcname}/releases/download/%{version}/%{name}-%{version}.tar.gz
+Source0: https://github.com/t0xic0der/%{srcname}/releases/download/v%{version}/%{name}-%{version}.tar.gz
 
 BuildArch: noarch
 
 BuildRequires: python3-devel
 BuildRequires: python3-setuptools
 
+Requires: python3-click
+Requires: python3-distro
```

Comment:
This is listed here

https://github.com/t0xic0der/nvidia-auto-installer-for-fedora/blob/45a703fc8741f05c1daad3d7c513d5c5bd334a02/setup.py#L35-L38

Any specific reason as to why we would want to have it here too?

Reply:
I missed it, it is reverted. thank you for pointing out.